docs/source/contributor-guide/index.md

```diff
@@ -106,6 +106,11 @@ A "major" PR means there is a substantial change in design or a change in the AP
 
 The good thing about open code and open development is that any issues in one change can almost always be fixed with a follow on PR.
 
+## Stale PRs
+
+Pull requests will be marked with a `stale` label after 60 days of inactivity and then closed 7 days after that.
```

Comment:
I'm thinking perhaps its good to also attach a link to `.github/workflows/stale.yml` to let the user check stale PR settings.... not sure if its super needed, but if lets say we change PR stale strategy in the yml but forget to change it in the contributor guide